src/pages/_document.tsx

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import React from 'react'
+// eslint-disable-next-line @next/next/no-document-import-in-page
```

Comment:
Was the build breaking without that?

Reply:
I was not able to commit because of this error. The 'yarn run lint' command was failing.

Reply:
Hmm... Seems to be something with your environment maybe. I just managed to commit [5e766bb](https://github.com/podkrepi-bg/frontend/pull/1355/commits/5e766bb632234eaf6525f74223b4aa0bd505c017)

Reply:
Yes, this is probably an issue related to my environment - other colleagues tested and they were able to commit too. I will handle it locally. Thanks for the commit! Lets see if it going to pass the build tests.